prettier.config.mjs

```diff
@@ -0,0 +1,8 @@
+/** @type {import('prettier').Config} */
+export default {
+  endOfLine: "lf",
+  semi: true,
+  singleQuote: false,
+  tabWidth: 2,
```

Comment:
Change to `4`

Reply:
That was one of the main reasons why I created this file, default tabWidth in prettier is 2 and is the consensus amongst Javascript code base.

![image](https://github.com/expressots/expressots/assets/43689101/e41a779e-d7c7-482a-a800-2521eeffcc2b)

Reply:
@Daniel-Boll, I understand what you are saying but let me give you my 2 cents on this.

When comes down to coding style there is no official documentation that supports the use of 4 or 2, neither camelCase, or PascalCase and so on. At the end of the day this is usually defined by the community without any official agreement, based on the majority of open source projects created with that particular style, creating an influence over the community, which produces usually a non official but more acceptable coding style. Also, companies internally, due to dealing with many different tech stacks, sometimes they tend to align their convention across languages as well, or create their own. In resume, all this boils down to personal preference. And that's what I am trying to avoid here. I am trying to make sure that if we decide a standard, we stick with that standard.

Another example that reinforces what I am saying is that, usually js, css is minified on production environment, even removing indentation out of the equation ;). In fact, this does not add any value but implies an organizational style to the code, which I don't see any problem. 

Approving this, we will be formatting our core code base, so that we will need to standardize these guys:
- CLI
- Prisma
- Templates
And everything else from now on. Does that make sense and is clear the impact of this little change at indentation?

Obs: The picture above doesn't sustain any evidence as it is from the prettier config recommendation and not from an official ecmascript or foundation recommendation.

Reply:
That is a valid point indeed, yet still another one would be that we are trying to adhere JS/TS contributors, and although you are correct, JS as many other languages didn't instituted specific naming patterns and etc, only language I can think of that has done that in a close to oficial tool would be Rust with Rust analyzer. But this lack of rules made the community pick some standards.

Another thing I was planning on changing is the casing in some variables and things. Our target is JS/TS developers having PascalCase variables, for instance, can be akin to the average user as it is seldom seen by the community.

This issue was born by the uncanny feeling I had looking at the code as an average JS/TS developer, which made me think that this was a valid point to be discussed.

Reply:
Regardless of the decision, we should be consistent across all repos indeed, perhaps the institution of precommit hook to ensure styling and linting would fit us well.

Reply:
Exactly, that's my main concern, consistency. You got it. I will be approving and merging to the main